app/controllers/apartments_controller.rb

```diff
@@ -1,2 +1,18 @@
 class ApartmentsController < ApplicationController
+  def index
+    apartments = Apartment.all
+    render json: apartments
+  end
+  def create
+   apartment = Apartment.create(apartment_params)
+    if apartment.valid?
+     render json: apartment
+    else
+     render json: apartment.errors, status: 422
+  end
 end
+private
+  def apartment_params
+    params.require(:apartment).permit(:street, :city, :state, :unit, :square_footage, :price, :bedrooms, :bathrooms, :pets, :image, :user_id)
+  end
```

Comment:
This should still be within the scope of the class.  The end on line 13 should be after your strong params

Reply:
making appropriate changes